modules/data.land/R/extract_soil_nc.R

```diff
@@ -237,6 +237,33 @@ extract_soil_gssurgo<-function(outdir, lat, lon, size=1, radius=500, depths=c(0.
   out.ense<-out.ense%>% 
     setNames(rep("path", length(out.ense)))
   
+  #connect to db 
+  # Extract info from settings and setup
+  site       <- settings$run$site
+  dbparms    <- settings$database
+  
+  bety <- dplyr::src_postgres(dbname   = dbparms$bety$dbname, 
+                              host     = dbparms$bety$host, 
+                              user     = dbparms$bety$user, 
+                              password = dbparms$bety$password)
+  con <- bety$con
+  # register files in DB 
+  for(i in 1:length(out.ense)){
+    in.path = paste0(dirname(out.ense[i]$path), '/')
+    in.prefix = stringr::str_remove(basename(out.ense[i]$path), ".nc")
+    
+    
+    dbfile.input.insert (in.path,
```

Comment:
I think we want to avoid putting the DB insert inside of the extract function. First, it violates our existing design decision to separate these processing operations from the DB inserts and makes it impossible to run the function remotely. Second, it goes against our recent discussion about making it easier to use PEcAn functions independent from the DB. The DB insert should be occurring in soil_process, which already has the settings object and an open con.